.github/workflows/copy-workflow.yml

```diff
@@ -31,47 +26,56 @@ jobs:
     - name: Checkout template repository
       uses: actions/checkout@v2
       with:
-        path: ${{ env.TARGET_DIR }}
-    - name: Add "DO NOT EDIT" header to ${{ env.FILE }}
+        path: ${{ env.TEMPLATE_REPO_DIR }}
+    - name: Create commits
       run: |
-        tmp=$(mktemp)
-        cat $TARGET_DIR/$TEMPLATE_DIR/header.yml $TARGET_DIR/$TEMPLATE_DIR/$FILE > $tmp
-        mv $tmp $TARGET_DIR/$TEMPLATE_DIR/$FILE
-    - name: Check if an update to ${{ env.FILE }} is neccessary
-      run: |
-        if [[ ! -f "${{ env.FILE }}" ]]; then
-          echo "First deployment of this file."
-          echo 'COMMIT_MSG=add ${{ env.FILE }}' >> $GITHUB_ENV
-          echo 'PR_TITLE=add ${{ env.FILE }}' >> $GITHUB_ENV
-          echo "NEEDS_UPDATE=1" >> $GITHUB_ENV
-          exit
-        fi
-        STATUS=$(cmp --silent $FILE $TARGET_DIR/$TEMPLATE_DIR/$FILE; echo $?)
-        if [[ $STATUS -ne 0 ]]; then
-          echo "Update needed."
-          echo "NEEDS_UPDATE=1" >> $GITHUB_ENV
-        else
-          echo "Files indentical. Skipping."
-        fi
-    - name: Copy ${{ env.FILE }} to target repo
-      if: ${{ env.NEEDS_UPDATE == 1 }}
-      run: |
-        dir=$(dirname $FILE)
-        mkdir -p $dir
-        cp $TARGET_DIR/$TEMPLATE_DIR/$FILE $dir
-        rm -rf $TARGET_DIR
+        git config user.name web3-bot
+        git config user.email "web3-bot@users.noreply.github.com"
+        for f in $(jq ".[]" <<< '${{ toJson(github.event.client_payload.files) }}' | xargs -n1); do 
```

Comment:
I think you're using the xargs to get rid of the quotes:

```
$ jq ".[]" <<< '["foo", "bar", "baz"]'
"foo"
"bar"
"baz"
$ jq ".[]" <<< '["foo", "bar", "baz"]' | xargs -n1
foo
bar
baz
```

But that's wrong, IMO - shell quoting is quite different to JSON quoting. Plus, jq already has an output flag for this, called `-r` for "raw output":

```
$ jq -r ".[]" <<< '["foo", "bar", "baz"]'
foo
bar
baz
```